expression/builtin_math_vec.go

```diff
@@ -223,3 +225,47 @@ func (b *builtinRoundDecSig) vecEvalDecimal(input *chunk.Chunk, result *chunk.Co
 func (b *builtinRoundDecSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinPowSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf1, err := b.bufAllocator.get(types.ETReal, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf1)
+	if err := b.args[0].VecEvalReal(b.ctx, input, buf1); err != nil {
```

Comment:
We can directly use the `result` variable to store the result of args[0]. Thus we can improve the performance by saving a memory copy.